xwiki-platform-core/xwiki-platform-flamingo/xwiki-platform-flamingo-skin/xwiki-platform-flamingo-skin-resources/src/main/resources/flamingo/less/buttons.less

```diff
@@ -9,72 +9,74 @@
   #gradient > .vertical-three-colors(lighten(@color, 10%), @color, 50%, darken(@color, 5%));
 }
 
+.btn-background(@color) {
+  background-color: @color;
+}
+
 .btn-border(@color) {
   border: 1px solid @color;
-  border-bottom-color: darken(@color, 10%);
-  .box-shadow(~"inset 0 1px 0 rgba(255,255,255,.2), 0 1px 2px rgba(0,0,0,.05)");
 }
 
 /* &-.button selectors are added in order to override the mapping and legacy usage of .button class */
 .btn {
   &-default {
-    .btn-gradient(lighten(@btn-default-bg, 5%));
-    .btn-border(@btn-default-bg);
+    .btn-background(lighten(@btn-default-bg, 5%));
+    .btn-border(darken(@btn-default-bg, 5%));
```

Comment:
I'm now using darken here because I was finding it hard to see the default button borders.